api/src/services/AuthService.ts

```diff
@@ -1,6 +1,10 @@
 import { inject, injectable } from 'inversify'
 import { EntityManager, Repository } from 'typeorm'
 import { User } from '../entities/User'
+import {validate} from "class-validator";
+
+const jwt = require('jsonwebtoken');
```

Comment:
Please move all bcryp and jsonwebtoken signing logic into a EncryptionService that is injected into this class. It should have encrypt(value) and sign(object) methods